src/libxtp/qmpackages/orca.cc

```diff
@@ -825,12 +825,51 @@ std::string Orca::WriteMethod() const {
   std::string convergence =
       this->_convergence_map.at(_settings.get("convergence_tightness")) +
       "SCF ";
-  stream << "! DFT " << _settings.get("functional") << " " << convergence
+  stream << "! DFT " << this->GetOrcaFunctionalName() << " " << convergence
          << opt
          // additional properties provided by the user
          << _settings.get("orca.method") << "\n";
   return stream.str();
 }
 
+std::string Orca::GetOrcaFunctionalName() const {
+
+  char* votca_share = getenv("VOTCASHARE");
+  if (votca_share == nullptr) {
+    return _settings.get("functional");
+  } else {
+    tools::Property all_functionals;
+
+    auto xml_file = std::string(getenv("VOTCASHARE")) +
+                    std::string("/xtp/packages/orca_functional_names.xml");
+
+    all_functionals.LoadFromXML(xml_file);
+
+    const tools::Property& functional_names =
+        all_functionals.get("functionals");
+
+    std::string input_name = _settings.get("functional");
+    // Some functionals have a composed named separated by a space
```

Comment:
@JensWehner  do you agree with this?

Reply:
what happens if votcashare is not set?

Reply:
Assuming that the user has command line tools in her/his path and the calculation doesn't fail because ti cannot find `xtp_tools`, `xtp_run`, etc.then that function will generate a call to Orca that will subsequently fails because it doesn't know the functional